.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Rust CI
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust:
+          - stable
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ matrix.rust }}
+        override: true
+
```

Comment:
I'd recommend adding a cache step here, simply:

```
- uses: Swatinem/rust-cache@v2
```

will speed up subsequent builds of this pipeline SIGNIFICANTLY

Reply:
I have added that as well! Thanks for the feedback. Let me know and if it looks good to you. The actions ran successfully.

Reply:
That change reduced build speed from 2m48s to 1m! amazing!